src/test/resources/data/create/yksilon-suosikki-test-data.sql

```diff
@@ -0,0 +1,128 @@
+INSERT INTO tyomahdollisuus_data.tyomahdollisuus
+(id, json_id)
+VALUES('00016334-886e-4d11-93f0-872fcf671920'::uuid, 'a20e0454-c949-42b9-9198-4da730a6c43d'::uuid);
+INSERT INTO tyomahdollisuus_data.tyomahdollisuus
+(id, json_id)
+VALUES('00143beb-0817-4e6d-9107-57d0245b57ee'::uuid, '51dbc87e-0261-473c-acfc-8cd20a945468'::uuid);
+INSERT INTO tyomahdollisuus_data.tyomahdollisuus
+(id, json_id)
+VALUES('0014885a-4aa6-4202-9865-2fcb4457cc59'::uuid, 'c8ee8856-68f1-4f28-a096-628298bd8134'::uuid);
+
+INSERT INTO tyomahdollisuus_data.tyomahdollisuus_kaannos
+(tyomahdollisuus_id, kaannos_key, otsikko, tiivistelma, kuvaus)
+VALUES('00016334-886e-4d11-93f0-872fcf671920'::uuid, 'SV', 'Muovituoteasentaja', 'Tässä on kahden lauseen yhteenveto muovituotantotyöntekijän tehtävistä:
+
+Muovituotantotyöntekijän tehtäviin kuuluu muovituotteiden valmistus, kuten muoviosien kokoonpano, muovihitsaus, maalaus ja pakkaus. Työ vaatii tarkkuutta, käsityötaitoja ja valmiutta vuorotyöhön.', 'Muovituotantotyöntekijä
+
+Muovituotantotyöntekijän tehtävät liittyvät muovituotteiden valmistukseen, kokoonpanoon ja laadunvalvontaan. Työtehtävät voivat sisältää esimerkiksi ruiskuvalukoneiden käyttöä, muottien vaihtoa, tuotteiden pakkaamista ja tarkastamista sekä muovihitsausta ja laminointia.
+
+Työpaikat sijaitsevat muoviteollisuuden yrityksissä eri puolilla Suomea. Esimerkkejä työnantajista ovat Coreplast, Orthex, MSK Plast, Prevex, Kaivotuote ja Talokaivo.
+
+Työ edellyttää usein valmiutta vuorotyöhön, kuten kolmivuorotyöhön. Työ voi olla fyysisesti raskasta ja edellyttää hyvää peruskuntoa. Tärkeiksi taidoiksi mainitaan näppäryys, tarkkuus, huolellisuus, oma-aloitteisuus ja oppimiskyky. Aikaisempi työkokemus muoviteollisuudesta tai vastaavista tehtävistä katsotaan eduksi, mutta ei ole välttämätöntä, sillä työhön perehdytetään.
+
+Työnantajat tarjoavat vakituisia ja määräaikaisia työsuhteita, joissa noudatetaan muoviteollisuuden työehtosopimusta. Palkkaus määräytyy työkokemuksen ja osaamisen mukaan, mutta on yleensä työehtosopimuksen mukainen tai hieman korkeampi.
+
+Muovituotantotyöntekijä, muovihitsaaja, kokoonpanija, operaattori, asentaja, maalaamotyöntekijä, raaka-ainevarastotyöntekijä, tuotantotyöntekijä.');
```

Comment:
Perhaps the test data could be simpler (no need to have the description)

Reply:
Data simplified.